relationships/candidates/MEMCACHEDCLUSTER.yml

```diff
@@ -1,10 +1,8 @@
-category: MEMCACHED
+category: MEMCACHEDCLUSTER
```

Comment:
I still don't see how this is supposed to match.
I am only seeing a `aws.endpoint` tag on the memcached cluster that you have provided.
Or maybe I am missing something ?

Reply:
My bad. Updated the cluster entity with correct tag names.